docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,280 @@
+| [`excludeFilter`](#exclude-headers)                                                 | Exclude specific headers when importing a library. Has priority over `headerFilter`.                                               |
+| [`strictEnums`](#configure-enums-generation)                                        | A space-separated list of enums that should be generated as a [Kotlin enum](enum-classes.md).                                      |
+| [`nonStrictEnums`](#configure-enums-generation)                                     | A space-separated list of enums that should be generated as integral values.                                                       |
+| [`noStringConversion`](#set-up-string-conversion)                                   | A space-separated lists of functions whose `const char*` parameters should not be auto-converted to Kotlin `String`s.              |
+| `depends`                                                                           |                                                                                                                                    |
+| `exportForwardDeclarations`                                                         |                                                                                                                                    |
+| `allowedOverloadsForCFunctions`                                                     |                                                                                                                                    |
+| [`disableDesignatedInitializerChecks`](#allow-calling-a-non-designated-initializer) | Disable the compiler check that doesn't allow calling a non-designated initializer as a `super()` constructor.                     |
+| [`foreignExceptionMode`](#wrap-exceptions)                                          | Wraps exceptions from Objective-C code into Kotlin exceptions with the `ForeignException` type.                                    |
+| `pluginName`                                                                        |                                                                                                                                    |
+| `objcClassesIncludingCategories`                                                    |                                                                                                                                    |
+| [`userSetupHint`](#linker-errors)                                                   | Adds a custom message, for example, to help users resolve linker errors.                                                           |
+
+In addition to the list of properties, you can include [custom declarations](#add-custom-declarations) in your definition
+file.
+
+### Headers
+
+When the library is imported, all of the headers are also included in the program. So all header dependencies are included
```

Comment:
"C library" is either a set of headers or a Clang module. So user have to either:
* list headers in `headers` property.
* specify a module in `modules` property.
* add custom declarations.
to make definition file meaningful.

So this section could be rewritten like
> If the library does not have a Clang module and represented as a set of headers, use `headers` property to specify headers that should be imported.